graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/internal/DgsDataLoaderRegistry.kt

```diff
@@ -165,34 +165,23 @@ open class DgsDataLoaderRegistry : DataLoaderRegistry() {
      * @return total number of entries that were dispatched from registered [org.dataloader.DataLoader]s.
      */
     override fun dispatchAllWithCount(): Int {
-        var sum = 0
-        scheduledDataLoaderRegistries.forEach {
-            sum += it.value.dispatchAllWithCount()
-        }
-        sum += dataLoaderRegistry.dispatchAllWithCount()
-        return sum
+        return scheduledDataLoaderRegistries.values.sumOf { registry -> registry.dispatchAllWithCount() } +
+            dataLoaders.values.sumOf { dataLoader -> dataLoader.dispatchWithCounts().keysCount }
     }
 
     /**
      * @return The sum of all batched key loads that need to be dispatched from all registered
      * [org.dataloader.DataLoader]s
      */
     override fun dispatchDepth(): Int {
-        var totalDispatchDepth = 0
-        scheduledDataLoaderRegistries.forEach {
-            totalDispatchDepth += it.value.dispatchDepth()
-        }
-        totalDispatchDepth += dataLoaderRegistry.dispatchDepth()
-
-        return totalDispatchDepth
+        return scheduledDataLoaderRegistries.values.sumOf { registry -> registry.dispatchDepth() } +
+            dataLoaders.values.sumOf { dataLoader -> dataLoader.dispatchDepth() }
```

Comment:
Same as above, can we use `super.dispatchDepth` which will give us the same result?